### PR TITLE
Some Dockerfile Tweaks and better wording in docs

### PR DESCRIPTION
**Pull request by alikhajeh1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-Redis in-memory data structure store, in a sentinel-based Master/Slave Replication setup, with persistent storage.
+Redis in-memory data store, in a sentinel-based Master/Slave Replication setup, with persistent storage.
 
 ## What is this?
 This template provides your project with a scalable sentinel-ready Redis Replication setup.
 It consists of a **Service**, **StatefulSet**, **Secret**, and a single initial master **Pod**.
 
-The way this replication works is that all redis pods (master & slaves) discover each other and decide which one should be the master. When master is down due to any reason, all the other slaves talk to each other using Sentinel to desice which one becomes master.
+The replication works by all Redis pods (master & slaves) discovering each other and deciding selecting a master. When the master is down for any reason, all the other slaves talk to each other using Sentinel and select a new master. When the old master comes back online, it joins as a slave.
```
Comment (alikhajeh1) [inline README.md]: I added `When the old master comes back online, it joins as a slave.` - is it true? have you tried it?

Comment (aramalipoor) [reply README.md]: I haven't tried it yet but from what I've read I didn't see anyone talking about such feature. If master joins he will be a slave. Democracy you know!

Comment (alikhajeh1) [reply README.md]: cool, give a shot so we can document it, it's the kind of thing that users will ask about if we don't

Comment (aramalipoor) [reply README.md]: I wanted to try it but unfortunately, i couldn't find a way to pause/unpause a pod. there was an `exec` to send STOP signal, but do u know any better way to temporarily make a pod unavailable?

Comment (alikhajeh1) [reply README.md]: What about just deleting the currently selected master pod?

Comment (aramalipoor) [reply README.md]: Good idea, in StatefulSets, pods will be created with the same name. If IP stays the same it can be good for this test, let me try.

**Pull request merged**